.travis.yml

```diff
@@ -56,6 +56,3 @@ before_script:
 
 before_install:
  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then nvm install 11.10.1 ; fi
-
-after_success:
```

Comment:
Just to make sure if this deletion is not a typo since you didn't mention it in the commit

Reply:
No officially removing codecov on purpose.